seeed/beaglev/flash.config

```diff
@@ -0,0 +1,2080 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Linux/x86 5.7.0 Kernel Configuration
```

Comment:
This isn't right because it's x86, but where should the flash.config come from?

Reply:
it's not right but ... this config is tested and works. Weird that it said that.

Reply:
I just copied this config from apu2, it is definitely wrong